<html>
<head>
<title>TestNG:  TC001_loginWithInvalidData</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>TC001_loginWithInvalidData</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>0/3/0</td>
</tr><tr>
<td>Started on:</td><td>Wed Sep 27 14:50:47 NZDT 2017</td>
</tr>
<tr><td>Total time:</td><td>23 seconds (23708 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest()'><b>loginWithInvalidDataTest</b><br>Test class: com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData<br/>
<a href="#Output-468487306" onClick='toggleBox("Output-468487306", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-468487306" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-468487306">
===============starting loginWithInvalidDataTest =================<br/>
clicking on signinButton<br/>
Entering invalid email address:- hello@gmail.com Object is :- [[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> xpath: .//*[@id='email']]<br/>
Entering invalid passoword:- 7621983719 Object is :-[[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> xpath: .//*[@id='passwd']]<br/>
clicking on submit button and Object is :- [[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> id: SubmitLogin]<br/>
<a href='D:\UIAutomation1\src\main\java\com\test\automation\UIAutomation1\screenshot\VerifyErrorMsg_27_09_2017_02_51_03.png'><img src='D:\UIAutomation1\src\main\java\com\test\automation\UIAutomation1\screenshot\VerifyErrorMsg_27_09_2017_02_51_03.png'height='100' width='100'/></a><br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: VerifyErrorMsg expected [true] but found [false]
	at com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest(TC001_loginWithInvalidData.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace468487306", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace468487306'><pre>java.lang.AssertionError: VerifyErrorMsg expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest(TC001_loginWithInvalidData.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>4</td>
<td>com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData@4cf777e8</td></tr>
<tr>
<td title='com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest()'><b>loginWithInvalidDataTest</b><br>Test class: com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData<br/>
<a href="#Output-412902837" onClick='toggleBox("Output-412902837", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-412902837" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-412902837">
===============starting loginWithInvalidDataTest =================<br/>
clicking on signinButton<br/>
Entering invalid email address:- hello@gmail.com Object is :- [[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> xpath: .//*[@id='email']]<br/>
Entering invalid passoword:- 7621983719 Object is :-[[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> xpath: .//*[@id='passwd']]<br/>
clicking on submit button and Object is :- [[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> id: SubmitLogin]<br/>
<a href='D:\UIAutomation1\src\main\java\com\test\automation\UIAutomation1\screenshot\VerifyErrorMsg_27_09_2017_02_51_07.png'><img src='D:\UIAutomation1\src\main\java\com\test\automation\UIAutomation1\screenshot\VerifyErrorMsg_27_09_2017_02_51_07.png'height='100' width='100'/></a><br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: VerifyErrorMsg expected [true] but found [false]
	at com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest(TC001_loginWithInvalidData.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace412902837", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace412902837'><pre>java.lang.AssertionError: VerifyErrorMsg expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest(TC001_loginWithInvalidData.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>3</td>
<td>com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData@4cf777e8</td></tr>
<tr>
<td title='com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest()'><b>loginWithInvalidDataTest</b><br>Test class: com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData<br/>
<a href="#Output-1748052403" onClick='toggleBox("Output-1748052403", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1748052403" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1748052403">
===============starting loginWithInvalidDataTest =================<br/>
clicking on signinButton<br/>
Entering invalid email address:- hello@gmail.com Object is :- [[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> xpath: .//*[@id='email']]<br/>
Entering invalid passoword:- 7621983719 Object is :-[[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> xpath: .//*[@id='passwd']]<br/>
clicking on submit button and Object is :- [[FirefoxDriver: firefox on ANY (3a03dd29-38b2-436b-bdac-31c84e8f7e00)] -> id: SubmitLogin]<br/>
<a href='D:\UIAutomation1\src\main\java\com\test\automation\UIAutomation1\screenshot\VerifyErrorMsg_27_09_2017_02_51_11.png'><img src='D:\UIAutomation1\src\main\java\com\test\automation\UIAutomation1\screenshot\VerifyErrorMsg_27_09_2017_02_51_11.png'height='100' width='100'/></a><br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: VerifyErrorMsg expected [true] but found [false]
	at com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest(TC001_loginWithInvalidData.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1748052403", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1748052403'><pre>java.lang.AssertionError: VerifyErrorMsg expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData.loginWithInvalidDataTest(TC001_loginWithInvalidData.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>3</td>
<td>com.test.autiomation.UIAutomation1.authenticationpage.TC001_loginWithInvalidData@4cf777e8</td></tr>
</table><p>
</body>
</html>